CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ This is the changelog for SpotBugs. This follows [Keep a Changelog v1.0.0](http:
 
 Currently the versioning policy of this project follows [Semantic Versioning v2.0.0](http://semver.org/spec/v2.0.0.html).
 
+## 3.1.13
```

Comment:
do not add version like this, instead, add entries under `## Unreleased`.

Reply:
I've switched to master branch and filled my changes to existing `## Unreleased` section